src/fileNamer.js

```diff
@@ -151,3 +151,139 @@ function mapValue(map, value) {
     return `-${map[value]}`
   }
 }
+
+function evaluateNewLabel(documents, L, LWP) {
```

Comment:
At first sight, I have trouble understanding what L and LWP are.

Reply:
These variables are defined and commented L68 in index.js when the 3 big functions are used.
```
 // TEMP LOGGING VAR NEED TO BE REMOVED                                                         
 const L = false //log without private data                                                     
 const LWP = false //log with private data (label can contain address) 
```
I wanted to keep it as short as possible to allow one line test and log when use in code. So that's why they are not very understanding as this.